frame/balances/src/lib.rs

```diff
@@ -362,7 +362,6 @@ pub mod pallet {
 
 	/// The total units issued in the system.
 	#[pallet::storage]
-	#[pallet::getter(fn total_issuance)]
```

Comment:
this is still removed?

Reply:
A method with the same name and return value is implemented in [Currency](https://github.com/paritytech/substrate/blob/3af83a501f9d50b523e795f6f89f9341a50bb889/frame/balances/src/impl_currency.rs#L220) and [fungible::Inspect](https://github.com/paritytech/substrate/blob/3af83a501f9d50b523e795f6f89f9341a50bb889/frame/balances/src/impl_fungible.rs#L29)